lib/auth/auth_with_roles.go

```diff
@@ -4927,6 +4927,6 @@ func verbsToReplaceResourceWithOrigin(stored types.ResourceWithOrigin) []string
 func publishAppChanges(server *Server, appType publisher.RemoteAppType) {
 	server.AppPublisher.Publish(publisher.AppChangeEvent{
 		AppType: appType,
-		Tenant:  server.GetTenantUrl(),
+		Tenant:  "",
```

Comment:
why is this empty for tenant? Can you add a comment if indeed it has to be empty

Reply:
Earlier we were passing the TenantUrl from the place where event is published, since TenantUrl is configuration, i have passed the tenantUrl as part of AppPublisherConfig that way it will be available to all the events published.

I have to pass the tenantUrl to all the places from where we publish the events.
Will add the comment or eliminate the empty value being passed here.

Reply:
You can eliminate the empty string. And then add the comment in the place where you keep the tenantUrl and mention  that this will be available for all events.